cli/src/main/resources/nls/Help.properties

```diff
@@ -34,6 +34,7 @@ cmd.mvn=Tool commandlet for Maven (Build-Tool).
 cmd.node=Tool commandlet for Node.js (JavaScript runtime).
 cmd.npm=Tool commandlet for Npm (JavaScript Node Package Manager).
 cmd.oc=Tool commandlet for Openshift CLI (Kubernetes management tool).
+cmd.pgadmin=Tool commandlet for pgAdmin.
```

Comment:
IMHO you created this feature branch from an outdated main branch.
Whenever you create a new feature branch, first do a pull of main from upstream.
With PR #389 you will be forced to also add `cmd.pgadmin.details`.